src/QMCDrivers/tests/QMCDriverNewTestWrapper.h

```diff
@@ -176,6 +176,20 @@ class QMCDriverNewTestWrapper : public QMCDriverNew
   };
 
   void testMeasureImbalance() { measureImbalance("Test"); }
+  void testDetermineStepsPerBlock()
+  {
+    CHECK(QMCDriverNew::determineStepsPerBlock(6, 36, 2, 3) == 2);
```

Comment:
Thanks for adding these. I think sufficient to avoid needing to add integration tests.